public/js/services/game.js

```diff
@@ -143,7 +143,15 @@ angular.module('mean.system')
       game.state = data.state;
     }
 
-    if (data.state === 'waiting for players to pick') {
+
+    if (data.state === 'black card') {
+      game.czar = data.czar;
+      if (game.czar === game.playerIndex) {
+        addToNotificationQueue('You are now the Czar, Click any of the black card to choose the question for the round.');
```

Comment:
When this text is been displayed on the browser, it's been covered by the black card. You can look for a better position to display the text so that it can be readable to the user.

Reply:
Fixed